Codeinwp/neve
# Starter sites and see all features upsells

### Summary
This pull is a combination of https://github.com/Codeinwp/neve/pull/3267 and https://github.com/Codeinwp/neve/pull/3261 with merge conflicts fixed

### Will affect visual aspect of the product

NO

### Screenshots 

Starter Sites card: https://vertis.d.pr/CucUAL
See all features: https://vertis.d.pr/Q2Q618

### Test instructions


- Make sure that clicking the "See all features" link goes to the appropriate section when themeisle.com page is loaded
- Upsell message and button inside starter sites card should appear whenever license is not active, not valid or Pro plugin not installed
- Upsell message and button inside starter sites card should not appear when valid license is set.


Closes Codeinwp/neve-pro-addon#1739
Closes Codeinwp/neve-pro-addon#1776
